src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -1,191 +1,197 @@
 package seedu.address.logic.parser;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static seedu.address.logic.parser.ParserUtil.MESSAGE_INVALID_INDEX;
-import static seedu.address.testutil.Assert.assertThrows;
-import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
-
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.Set;
 
 import org.junit.jupiter.api.Test;
 
+import seedu.address.model.module.Description;
+import seedu.address.model.module.Grade;
+import seedu.address.model.module.ModularCredit;
+import seedu.address.model.module.ModuleCode;
+import seedu.address.model.module.ModuleName;
+import seedu.address.model.module.Semester;
+import seedu.address.model.module.Year;
+import static seedu.address.testutil.Assert.assertThrows;
```

Comment:
Seems to be incorrect import order. Static imports should be above. Can run gradle's checkstyletest to verify.